<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Cadastro_Setores</class>
 <widget class="QWidget" name="Form_Cadastro_Setores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1009</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1009</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Setores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/add_sectors.gif</normaloff>../Images/add_sectors.gif</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_setores_gif">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/add_sectors.gif</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cadastro_setores_main">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>36</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cadastro de Setores</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_cod_setor">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Nome Setor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_cod_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="placeholderText">
        <string>Insira o Nome do Setor...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_desc_setores">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_desc_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="placeholderText">
        <string>Insira a Descrição do setor...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_responsavel_setores">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Responsável:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_responsavel_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="placeholderText">
        <string>Insira o Responsável pelo setor...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_localizacao_setores">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Localização do Setor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_localizacao_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="documentTitle">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Escreva aqui as informações extra sobre a localização do setor... (até 225 caracteres)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_obs_setores">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Observações:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_obs_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="documentTitle">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Escreva aqui as informações extra sobre o setor... (até 225 caracteres)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pushButton_cadastrar_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CADASTRAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_limpar_tela_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LIMPAR TELA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sair_setores">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SAIR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_feedback_setores">
     <property name="font">
      <font>
       <family>Sitka Heading</family>
       <pointsize>25</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,0,0);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhUppercaseOnly</set>
     </property>
     <property name="text">
      <string>FEEDBACK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
